index.bs

```diff
@@ -385,6 +414,115 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
         {{VideoConfiguration/contentType}}, if possible. Otherwise, the UA should
         presume that the video stream cannot produce alpha channel information.
       </p>
+
+      <p>
+        <pre class='idl'>
+          enum HdrMetadataType {
+            "smpteSt2086",
+            "smpteSt2094-10",
+            "smpteSt2094-40"
+          };
+        </pre>
+
+        <p>
+          If specified, the <dfn for='VideoConfiguration' dict-member>hdrMetadataType</dfn> 
+          represents the HDR metadata type the decoder needs to support in order
+          to successfully decode the video track.
+        </p>
```

Comment:
HDR metadata typically includes descriptive information about the luminance / color ranges present in the encoded video and other information about creative intent that is valuable to improve the rendering of the video on devices that do not have the capability to accurately render those luminance / color ranges. As such, support of the metadata is not necessary to "decode the video track" - or even to render it in some cases depending on device capability.

I'm not sure exactly what is the best language as the pattern here seems to be say that these things describe something "supported by the decoder". But aren't we *asking* what is supported here ?

Perhaps we should describe the capability identified by each enum, for example, here we would say 'the hdrMetadataType describes the capability to interpret HDR metadata of the specified type.'

Reply:
Thanks for the suggestion. That sounds good.